src/index.js

```diff
@@ -1,59 +1,84 @@
-import invariant from 'invariant';
+// @flow
 /*
-This is pretty simplistic at the moment, since it doesn't handle references. More work is needed to actually
+  This is pretty simplistic at the moment, since it doesn't handle references.
+  More work is needed
 */
+import invariant from 'invariant';
+
+// Global types
+declare var MSJSONDataArchiver: Object;
+declare var MSJSONDictionaryUnarchiver: Object;
+declare var NSBundle: Object;
+type SketchObject = any;
 
 /*
 Versions based on discussion info: http://sketchplugins.com/d/316-sketch-version
 */
-// Internal Sketch Version (ex: 95 => v47 and below)
-const SKETCH_HIGHEST_COMPATIBLE_VERSION = '95';
+// Internal Sketch Version
+// Using a high number to basically allow all versions
+const SKETCH_HIGHEST_COMPATIBLE_VERSION = '999';
```

Comment:
Could you separate the changes related to the default version from the config update (which looks fine)?

There seems to be an ongoing confusion about the purpose of having an explicitly defined version number: perhaps we can chat in real-time to discuss?

Reply:
Hey yeah, can separate them if need be. Otherwise happy to discuss the version stuff in real time. Feel free to DM me and we set something up 👍